<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoystickDialog</class>
 <widget class="QDialog" name="JoystickDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configure Joystick</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ch2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSlider" name="Ch_3">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32676</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="jsmapCB_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Not  Assigned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P3/LS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ch1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="jsmapCB_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Not  Assigned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P3/LS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="Ch_2">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32676</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="jsmapCB_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Not  Assigned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P3/LS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ch4</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="Ch_5">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32676</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Ch6</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ch3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QComboBox" name="jsmapCB_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Not  Assigned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P3/LS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSlider" name="Ch_4">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32676</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QComboBox" name="jsmapCB_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Not  Assigned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P3/LS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="Ch_6">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32676</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QComboBox" name="jsmapCB_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Not  Assigned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P3/LS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Ch5</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Ch7</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSlider" name="Ch_7">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32676</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QComboBox" name="jsmapCB_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Not  Assigned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P3/LS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Ch8</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="Ch_8">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32676</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QComboBox" name="jsmapCB_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Not  Assigned</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Vertical</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Left Horizontal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>P3/LS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="Ch_1">
         <property name="minimum">
          <number>-32767</number>
         </property>
         <property name="maximum">
          <number>32676</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="ChInv_1">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="ChInv_2">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="ChInv_3">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="ChInv_4">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="ChInv_5">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="ChInv_6">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="ChInv_7">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="ChInv_8">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="howtoLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>40</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(255, 255, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Instructions</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>198</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Ch_1</tabstop>
  <tabstop>ChInv_1</tabstop>
  <tabstop>jsmapCB_1</tabstop>
  <tabstop>Ch_2</tabstop>
  <tabstop>ChInv_2</tabstop>
  <tabstop>jsmapCB_2</tabstop>
  <tabstop>Ch_3</tabstop>
  <tabstop>ChInv_3</tabstop>
  <tabstop>jsmapCB_3</tabstop>
  <tabstop>Ch_4</tabstop>
  <tabstop>ChInv_4</tabstop>
  <tabstop>jsmapCB_4</tabstop>
  <tabstop>Ch_5</tabstop>
  <tabstop>ChInv_5</tabstop>
  <tabstop>jsmapCB_5</tabstop>
  <tabstop>Ch_6</tabstop>
  <tabstop>ChInv_6</tabstop>
  <tabstop>jsmapCB_6</tabstop>
  <tabstop>Ch_7</tabstop>
  <tabstop>ChInv_7</tabstop>
  <tabstop>jsmapCB_7</tabstop>
  <tabstop>Ch_8</tabstop>
  <tabstop>ChInv_8</tabstop>
  <tabstop>jsmapCB_8</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>nextButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources>
  <include location="companion.qrc"/>
 </resources>
 <connections/>
</ui>
